### PR TITLE
Add enum

### PR DESCRIPTION
**Pull request by jklmnn**
ref #24

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,6 +181,14 @@ function Bar (N : Natural) return Natural is
       Convention => C;
 ```
 
+```Ada
+type Enum is
+   (Very_Very_Very_Very_Long_Literal_1 => 1,
+    Very_Very_Very_Very_Long_Literal_2 => 2,
+    Very_Very_Very_Very_Long_Literal_3 => 3)
```
Comment (senier) [inline README.md]: Maybe we want to shorten one of the very very very very long literals to show that arrows are supposed to be aligned?

File: README.md [hunk 2]
```diff
@@ -181,6 +181,14 @@ function Bar (N : Natural) return Natural is
...
+with Aspect => Condition;
+```
```
Comment (treiher) [inline README.md]: Does this style only apply for very long literals or for all cases? Do we want to allow single line enums?

```Ada
type Enum is (A => 1, B => 2, C => 3) with Aspect => Condition;
```

Comment (jklmnn) [reply README.md]: I added a comment to this in #24 since there are more implications to this.

**Review by senier — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jklmnn) [reply README.md]: Fixed in https://github.com/Componolit/ada-style/pull/25/commits/2d9a52028ec9141402319880d6242abc794eedf0. There aren't any arrows anymore.

Comment (senier) [reply README.md]: Fair enough, then I think we don't have to align them ;)

**Review by senier — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jklmnn) [reply README.md]: Fixed in https://github.com/Componolit/ada-style/pull/25/commits/2d9a52028ec9141402319880d6242abc794eedf0.

**Review by treiher — APPROVED** [review]

**Pull request merged**